angular/src/directives/control-value-accessors/value-accessor.ts

```diff
@@ -96,13 +96,11 @@ export class ValueAccessor implements ControlValueAccessor, AfterViewInit, OnDes
     if (formControl) {
       const methodsToPatch = ['markAsTouched', 'markAllAsTouched', 'markAsUntouched', 'markAsDirty', 'markAsPristine'];
       methodsToPatch.forEach((method) => {
-        if (formControl.get(method)) {
```

Comment:
Why does removing the `if` fix things here?

Reply:
So we should have an if-check (added a new commit), but one that checks if the method is defined on the form control before overwriting its behavior.

With the [commit](https://github.com/ionic-team/ionic-framework/pull/23970/files#diff-6463538dcd7093e03dd68d67578c21c1068a223af315e14e37069c5f1507132e) to upgrade to Angular 12, I believe it was unintentionally modified.

The `control#get` API is used for accessing form controls on a form group, not for accessing the mark methods.

Reply:
Makes sense. thanks!